roles/kubernetes-apps/ansible/tasks/nodelocaldns.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Kubernetes Apps | Lay Down nodelocaldns Template
+  template:
+    src: "{{ item.file }}.j2"
+    dest: "{{ kube_config_dir }}/{{ item.file }}"
+  with_items:
+    - { name: nodelocaldns, file: nodelocaldns-config.yml, type: configmap }
+    - { name: nodelocaldns, file: nodelocaldns-sa.yml, type: sa }
+    - { name: nodelocaldns, file: nodelocaldns-deamonset.yml, type: daemonset }
+  register: nodelocaldns_manifests
+  vars:
+    clusterIP: "{{ skydns_server }}"
```

Comment:
How does this work with `coredns_dual` deployment, where you have two DNS servers deployed?

Reply:
Hm will need to give that an extra thought.

Reply:
I think I got it.